module/zfs/zfs_replay.c

```diff
@@ -361,6 +361,7 @@ zfs_replay_create_acl(zfs_sb_t *zsb, lr_acl_create_t *lracl, boolean_t byteswap)
 			    lr->lr_uid, lr->lr_gid);
 		}
 
+		VERIFY(name != NULL);
```

Comment:
In both of these cases it would be best to gracefully detect the NULL and handle it.  It's plausible that a bogus ZIL entry with a somehow correct checksum could encounter this.  This could render the pool unimportable due to the VERIFY or previous to this change because of a panic.  I'd suggest setting `error = SET_ERROR(EINVAL);` going exiting through the bail label.